### DISCUSSION TITLE
Apache Airflow with integrated IDE

### DISCUSSION THREAD

**Discussion by vmellos** [apache/airflow#25076] [Ideas]
Nowadays the airflow UI allows us to see the code of the dag, I think having an option of editing the code quickly on Apache Airflow in debug mode I suppose, through the integrated IDE makes sense.

**Comment by potiuk** [comment]
That's not possible. Webserver has no access to write the code and this security feature, not a bug.

 What's wrong with editing the code in "real IDE" ? IntelliJ? PyuCharm? VSCode of your choice? If you keep your code in GitHub you can edit them with Github VSCode integration directly in the browser. 

I think code editing for DAGS should be "property of DAG storage" not Airlfow webserver which is mostly an "operational" interface.
